### PR TITLE
Balder/gc legacy bucketmove and lidspace jobs

### PR DESCRIPTION
**Pull request by baldersheim**
@toregge @geirst PR

### CODE REVIEW

**Review by toregge — APPROVED** [review]

File: searchcore/src/tests/proton/documentdb/maintenancecontroller/maintenancecontroller_test.cpp [hunk 1]
```diff
@@ -929,57 +929,6 @@ MaintenanceControllerFixture::removeDocs(const test::UserDocuments &docs, Timest
     }
 }
 
-TEST_F("require that bucket move controller is active", MaintenanceControllerFixture)
```
Comment (geirst) [inline searchcore/src/tests/proton/documentdb/maintenancecontroller/maintenancecontroller_test.cpp:932]: @baldersheim Is a similar test relevant for bucket mover v2?

File: searchcore/src/tests/proton/documentdb/maintenancecontroller/maintenancecontroller_test.cpp [hunk 2]
```diff
@@ -1054,74 +1003,6 @@ TEST_F("require that periodic session prunings are scheduled",
     ASSERT_TRUE(f._gsp.isInvoked);
 }
 
-TEST_F("require that active bucket is not moved until de-activated", MaintenanceControllerFixture)
```
Comment (geirst) [inline searchcore/src/tests/proton/documentdb/maintenancecontroller/maintenancecontroller_test.cpp:1057]: @baldersheim Is a similar test relevant for bucket mover v2?

Comment (baldersheim) [reply searchcore/src/tests/proton/documentdb/maintenancecontroller/maintenancecontroller_test.cpp:1057]: The bucket move job is tested thoroughly in documentbucketmover_test.cpp. So I think this one is not necessary.

File: searchcore/src/tests/proton/documentdb/maintenancecontroller/maintenancecontroller_test.cpp (hunk 1 above)
Comment (baldersheim) [reply searchcore/src/tests/proton/documentdb/maintenancecontroller/maintenancecontroller_test.cpp:932]: The bucket move job is tested thoroughly in documentbucketmover_test.cpp. So I think this one is not necessary.

**Pull request merged**